frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -198,12 +198,16 @@ const SearchField: React.FC = () => {
   const [showOptions, setShowOptions] = React.useState(false);
   const [open, setOpen] = React.useState(false);
 
-  const renderResult = (option: SearchIndex) => {
+  const renderResult = (props, option: SearchIndex) => {
     const handleSelection = () => {
       navigate(option.path);
     };
 
-    return <Result option={option} handleSelection={handleSelection} />;
+    return (
+      <li {...props}>
```

Comment:
for my understanding, why is the `<li>` element needed?

Reply:
https://mui.com/material-ui/migration/v5-component-changes/#update-renderoption

It used to be wrapped around the render options by default but they removed it in muiv5, this puts it back to keep parity for the moment.